ca_gazebo/launch/create_april_tags.launch

```diff
@@ -0,0 +1,34 @@
+<launch>
+	<arg name="env" default="april_tags"/>
+ 	<param name="create1/amcl/initial_pose_x" value="0"/>
+  	<param name="create1/amcl/initial_pose_y" value="0"/>
+  	<param name="create1/amcl/initial_pose_a" value="0"/>
+
+ 	<param name="create2/amcl/initial_pose_x" value="0"/>
+  	<param name="create2/amcl/initial_pose_y" value="0"/>
+  	<param name="create2/amcl/initial_pose_a" value="0"/>
+
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+	
+
+	<!-- Launch Gazebo world -->
```

Comment:
The other launch files don't have this approach. Why did you include these lines here?

Reply:
You're right. Now I changed it so it follows the rest of the launch files structure. It's more simple that way.